subsys/bluetooth/controller/ll_sw/ull_scan_aux.c

```diff
@@ -1280,7 +1275,16 @@ static void flush(void *param)
 	scan = HDR_LLL2ULL(lll);
 	scan = ull_scan_is_valid_get(scan);
 	if (!IS_ENABLED(CONFIG_BT_CTLR_SYNC_PERIODIC) || scan) {
+		/* TODO: Would the reset of Scan context association with Aux
+		 * context here interfere with LLL scheduling? May be LLL
+		 * scheduling will gracefully stop further reception and release
+		 * linked Rx buffers maintained in the Aux context?
+		 *
+		 * Is this code redundant/faulty here, now that Scan LLL and Aux
+		 * ULL scheduling reset it anyway?
+		 */
```

Comment:
Who are you asking? :D

Reply:
This code has been here, and has not caused harm yet :-) .... the comment is here so that in case we faced the scenario that the question ask, then we know that this is the place to look for. CI is passing and a long duration manual test of running passive scanning with over 10 extended advertiser of ZLL and SDC application around seems stable for over couple hours.
Do not want to delete this code yet, unless i can get a failure due to this code. git blame will associate this comment with this commit and will help with thought process in case of future failure.